package gamer.protocol;

message CreateRoomMsgProtocol {
  required int32 room_owner_id  	= 1;
  required int32 rounds_num  		= 2;
  required int32 players_num  		= 3;
  optional int32 room_cards_num 	= 4[default = 0];
  optional int32 room_id  		    = 5[default = 0];
}